packages/api/lib/granule-remove-from-cmr.js

```diff
@@ -51,12 +55,20 @@ const unpublishGranule = async ({
   pgCollection,
   granulePgModel = new GranulePgModel(),
   granuleDynamoModel = new models.Granule(),
+  removeGranuleFromCmrFunction = _removeGranuleFromCmr(),
 }) => {
   // If we cannot find a Postgres Collection or Postgres Granule,
   // don't update the Postgres Granule, continue to update the Dynamo granule
   const pgGranuleCumulusId = pgGranuleRecord.cumulus_id;
   let dynamoGranuleDeleted = false;
   try {
+    let collectionId;
```

Comment:
```suggestion
    /** @type {string} */
    let collectionId;
```

Reply:
Still feel we should have a discussion re: the effort impact of this type annotation versus just doing it right in TS.        Obviously don't disagree with the intent. 

02b69e09bb